js/page-specific/swiftype-search-results.js

```diff
@@ -0,0 +1,91 @@
+namespacer('baltimoreCounty.pageSpecific');
+
+baltimoreCounty.pageSpecific.swiftypeSearchResults = (function swiftypeSearchResults($,
+	querystringer, Handlebars) {
+	var $searchResultsTarget;
+	var templateSelector = '#swiftype-search-results-template';
+	var errorMessageHtml = '<p>There were no results found for this search.</p>';
+
+	function getSearchResults(searchTerm, pageNumber) {
+		var currentPageNumber = pageNumber || 1;
+		var cleanedSearchTerm = searchTerm.replace(/\+/g, '%20');
+
+		$.ajax('//ba224964/api/search/' + cleanedSearchTerm + '/' + currentPageNumber).then(searchResultRequestSuccessHandler, searchResultRequestErrorHandler);
+	}
+
+	function searchResultRequestErrorHandler(err) {
+		console.log(err);
+	}
+
+	function searchResultRequestSuccessHandler(response) {
```

Comment:
I think this could be broken down into smaller functions.

Reply:
I moved things around a little, but considering the release is tomorrow, I don't have the time to heavily refactor.